testApp/src/Getters.tsx

```diff
@@ -129,7 +129,17 @@ export default function Getters() {
       <Getter
         name="getUserStatus vendors"
         call={async () => {
-          return await (await Didomi.getUserStatus()).vendors;
+          return (await Didomi.getUserStatus()).vendors;
+        }}
+        test={() => {
+          return true;
+        }}
+      />
+
+      <Getter
+        name="getUserStatus vendors globalConsent"
```

Comment:
Same, getter name could be global_consent

Reply:
change pushed